src/app/screens/Home/index.tsx

```diff
@@ -30,9 +32,13 @@ function Home() {
   const [isBlocked, setIsBlocked] = useState<boolean>(false);
   const [currentUrl, setCurrentUrl] = useState<URL | null>(null);
   const [payments, setPayments] = useState<Transaction[]>([]);
+  const [invoiceTransactions, setInvoiceTransactions] = useState<
```

Comment:
this sounds a bit doppeltgemoppelt.

and we do `invoiceTransactions(invoices)`

Reply:
Done